modules/oneko/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, pkgs, config, ... }:
+with lib;
+let cfg = config.mayniklas.oneko;
+in
+{
+  options.mayniklas.oneko = {
+    enable = mkEnableOption "activate oneko";
+  };
+  config = mkIf cfg.enable {
+    environment.systemPackages = with pkgs; [ oneko ];
+    systemd.user.services.oneko = {
+      description = "oneko";
+      serviceConfig.PassEnvironment = "DISPLAY";
+      serviceConfig.ExecStart = "oneko -sakura";
```

Otherwise the service won't find oneko.
```suggestion
      serviceConfig.ExecStart = "${pkgs.oneko}/bin/oneko -sakura";
```